tests/ui/containers/__snapshots__/entropy.test.tsx.snap

```diff
@@ -12,7 +12,7 @@ Object {
 exports[`Entropy container correctly handles added points with the entropy generator 2`] = `
 Object {
   "encodedEntropy": "",
-  "entropyProgress": 1.6666666666666667,
+  "entropyProgress": 0,
```

Comment:
This snapshot is now identical to the previous one (before a point has been added). Is this the intention?

Reply:
No not intentional, I think no need to check two snapshots. Probably made sense at some point before I changed the entropy container.